ashly/Ashly.cs

```diff
@@ -61,10 +62,10 @@ public class Ashly {
             Machines.Call createLobby = () => {
                 client.CreateLobby("cow");
             };
-            Machines.Call joinRadiant = () => {
+            /*Machines.Call joinRadiant = () => {
```

Comment:
Just curious does this change the behavior of the program? I'm not necessarily opposed to changing it but would like to understand. I like the idea of the example actually joining as a player but open to alternatives.

In general probably better to just delete code (here and everywhere) instead of commenting it out.

Reply:
Ah just saw your update in the README. Just curious what does it mean to not get the correct IP address in this context? Is it the hand-off from the lobby to the game server is broken because the game server IP isn't included?

Reply:
Somehow bot join radiant side without this comman, so i decided to comment it

Reply:
> Ah just saw your update in the README. Just curious what does it mean to not get the correct IP address in this context? Is it the hand-off from the lobby to the game server is broken because the game server IP isn't included?

Idk how it worked in 2015, but now field connect have this data A:1:4165184530:25334
I believed that this 4165184530 can be the IP of server, but actually not.
I ran server with 2 accounts (one from code, one from dota client) and compare packages, packages from client was sent to one IP, packages from code to another. After that i just give up

Reply:
> In general probably better to just delete code (here and everywhere) instead of commenting it out.

Not sure, this can be used as in code documentation :)

But if you really want, i can delete this